blpapijs.cpp

```diff
@@ -60,16 +60,36 @@
 #endif
 
 #define BLPAPI_EXCEPTION_TRY try {
-#define BLPAPI_EXCEPTION_CATCH                                              \
-    } catch (blpapi::Exception& e) {                                        \
-        NoRetThrowException(Exception::Error(                               \
-            NEW_STRING(e.description().c_str())));                          \
+
+#define BLPAPI_EXCEPTION_NEW(type)                                                         \
+    Local<Object> err = Exception::Error(NEW_STRING(e.description().c_str()))->ToObject(); \
```

Comment:
Break this up on multiple lines to keep everything within 80 cols (backslashes should line up).